protocol/avdl/device.avdl

```diff
@@ -16,6 +16,7 @@ protocol device {
     // not possible to determine revoker without major code change
```

Comment:
Ugh I had removed that earlier, but reverted and redid the change and forgot to remove again. Sorry!